frontend/src/__tests__/cypress/cypress/tests/e2e/pipelines/pipelines.cy.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  createOpenShiftProject,
+  deleteOpenShiftProject,
+} from '~/__tests__/cypress/cypress/utils/oc_commands/project';
```

Comment:
Is it common to use `~` when importing packages?

Reply:
yes, it works as a base directory